src/ports/postgres/modules/deep_learning/madlib_keras_automl.py_in

```diff
@@ -158,11 +174,10 @@ class HyperbandSchedule():
                                       **locals())
             plpy.execute(insert_query)
 
-@MinWarning("warning")
-class KerasAutoML():
-    """The core AutoML function for running AutoML algorithms such as Hyperband.
-    This function executes the hyperband rounds 'diagonally' to evaluate multiple configurations together
-    and leverage the compute power of MPP databases such as Greenplum.
+# @MinWarning("warning")
```

Comment:
Should we revert these commented lines back?

Reply:
They error out if I do that:
```
ERROR:  TypeError: Error when calling the metaclass bases
    function() argument 1 must be code, not str (plpy_elog.c:121)
```